packages/palette/src/elements/Button/Button.story.tsx

```diff
@@ -155,3 +156,23 @@ export const As = () => {
     </States>
   )
 }
+
+export const WithIcon = () => {
+  return (
+    <States<ButtonProps>
+      states={[
+        {},
+        { focus: true },
+        { hover: true },
+        { loading: true },
+        { disabled: true },
+        { loading: true, disabled: true },
+      ]}
+    >
+      <Button variant="secondaryOutline" size="small">
+        <BellIcon fill="currentColor" mr={0.5} />
+        Create an Alert
+      </Button>
+    </States>
+  )
+}
```

Comment:
Its slightly annoying to have to do this in two places, but additions like this would benefit being in our formal docs. Reminder that our docs exist and more docs = better team.